deploy/examples/cluster-multus-test.yaml

```diff
@@ -1,49 +0,0 @@
-#################################################################################################################
```

Comment:
The only multus examples are in the docs now? What about updating this example to match the doc example?

Reply:
This was an accidental deletion. The file **should** be back now, but let me know if you see it disappear again. I have been having trouble getting it to stay put for some reason